sdk/eventhub/azure-eventhub/samples/sync_samples/send_and_receive_amqp_annotated_message.py

```diff
@@ -17,6 +17,16 @@
 CONNECTION_STR = os.environ['EVENT_HUB_CONN_STR']
 EVENTHUB_NAME = os.environ['EVENT_HUB_NAME']
 
+import logging
+import sys
+handler = logging.StreamHandler(stream=sys.stdout)
+logger = logging.getLogger('azure.eventhub')
+logger.setLevel(logging.DEBUG)
+logger.addHandler(handler)
```

Comment:
all the debug statements and commented out code will need to be reset :)

Reply:
oops - good catch 😅